### DISCUSSION TITLE
SSH_AUTH_SOCK while building

### DISCUSSION THREAD

**Discussion by rjcortese** [lima-vm/lima#684] [Q&A] [answered]
### Description

Hello~

`limactl version 0.8.3` on MacOS 12.1

I want to use the `--ssh` flag while building an image like:
`nerdctl.lima build --ssh default .`
but I don't have `SSH_AUTH_SOCK` set in the lima vm.

Any ideas?

Thanks!

**Comment by jandubois** [comment, marked as answer]
Did you set `ssh.forwardAgent` to `true` in your `lima.yaml`?

```yaml
ssh:
  forwardAgent: true
```

See https://github.com/lima-vm/lima/blob/01ce318b113b88196ad4862278fdb9876cb03cc8/pkg/limayaml/default.yaml#L68-L80

**Reply by rjcortese** [reply]
Thank you! I didn't know about this config option!
